raccoglitesi.go

```diff
@@ -31,36 +31,36 @@ type Dipartimento struct {
 	code string
 }
 
-// descrive una sezione di tesi per un singolo professore
-type Sezione[Contenuto any] struct {
-	titolo   string
-	elementi []Contenuto
-}
-
+// Docente descrive un docente con le sue tesi
 type Docente struct {
 	nome  string
 	ruolo string
 	url   string
 	tesi  []Sezione[Sezione[string]]
 }
 
+// Sezione descrive una sezione con un titolo e una lista di elementi di tipo Contenuto
+type Sezione[Contenuto any] struct {
+	titolo   string
+	elementi []Contenuto
+}
+
 func getTesiURL(baseURL string) string {
-	return baseURL + TAB_TESI_SUFFIX
+	return baseURL + tabTesiSuffix
 }
 
 func collyVisit(r *colly.Request) {
 	if !*quiet {
-		log.Println("Visiting", r.URL.String())
+		fmt.Println("Visiting", r.URL.String())
 	}
 }
 
 func collyError(r *colly.Response, err error) {
-	log.Println("Request URL:", r.Request.URL, "failed with response:", r.StatusCode, "\nError:", err)
+	fmt.Fprintln(os.Stderr, "Request URL:", r.Request.URL, "failed with response:", r.StatusCode, "\nError:", err)
```

Comment:
I messaggi sono in due lingue diverse (sì, so che era già così).
Magari sceglierne una delle due.